File: Algoritmos Gulosos/G.py
import sys

input = sys.stdin.readline

def solve(participantes):
    maior = 0
    for i in range(0, len(participantes), 2):
        if participantes[i+1] - participantes[i] > maior:
            maior = participantes[i+1] - participantes[i]

    return maior

t = int(input())

for _ in range(t):
    n = int(input())
    participantes = list(map(int, input().split()))
    participantes.sort()
    print(solve(participantes))

File: Algoritmos Gulosos/G.java
import java.io.*;
import java.util.*;

class G{
    public static int solve(int[] participantes){
        int max = 0;
        for(int i = 0; i < participantes.length; i += 2){
            max = Math.max(max, Math.abs(participantes[i] - participantes[i+1]));
        }
        return max;
    }
    public static void main(String[] args) throws Exception{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(br.readLine());

        for(int i = 0; i < t; i++){

            int n = Integer.parseInt(br.readLine());
            int[] participantes = new int[n];
            String linha = br.readLine();
            StringTokenizer st = new StringTokenizer(linha);
            for(int j = 0; j < n; j++){
                participantes[j] = Integer.parseInt(st.nextToken());
            }
            Arrays.sort(participantes);
            System.out.println(solve(participantes));
        }
    }
}